src/main/pg/pg_ids.h

```diff
@@ -78,6 +78,7 @@
 #define PG_SERVO_CONFIG 52
 #define PG_IBUS_TELEMETRY_CONFIG 53 // CF 1.x
 //#define PG_VTX_CONFIG 54 // CF 1.x
+#define PG_TIMER_IO_CONFIG 55 // used to store the index for timer use in timerHardware array in target.c
```

Comment:
Not sure if iNav wants to adopt this. Should maybe go into the Betaflight range.